deploy.sh

```diff
@@ -118,7 +118,7 @@ function terraform_init {
   fi
 
   local ROBOT_IMAGE_DIGEST
-  ROBOT_IMAGE_DIGEST=$(cat bazel-bin/src/go/cmd/setup-robot/setup-robot.push.digest)
+  ROBOT_IMAGE_DIGEST=$(cat bazel-bin/setup-robot.digest)
```

Comment:
Also need to change line 86 and replace --dst with --repository and --tag. Same for .github/ci/common.sh:99